app/Http/Requests/PendudukRequest.php

```diff
@@ -80,10 +80,9 @@ public function rules()
             "penduduk.*.tahun"                 => "required|int",
             "penduduk.*.created_at"            => "required|date",
             "penduduk.*.updated_at"            => "required|date",
-            "penduduk.*.id_pend_desa"          => "required|integer",
 
             // Batch delete validation
-            "hapus_penduduk.*.nik" => "present|min:16|max:16",
+            "hapus_penduduk.*.id_pend_desa" => "present|integer",
```

Comment:
Bisa minta tolong tambahkan satu lagi validasi request pak, untuk mengecek desa_id
```suggestion
            "hapus_penduduk.*.id_pend_desa" => "present|integer",
            "hapus_penduduk.*.desa_id" => "present|string|exists:das_data_desa,desa_id",
```

Reply:
@agungsugiarto Terimakasih, desa_id sudah ditambahkan ke validasi request hapus